crypto/mem_sec.c

```diff
@@ -446,15 +457,19 @@ static int sh_init(size_t size, size_t minsize)
         else
             pgsize = (size_t)tmppgsize;
     }
+#elif defined(_WIN32)
+    GetSystemInfo(&systemInfo);
+    pgsize = (size_t)systemInfo.dwPageSize;
 #else
     pgsize = PAGE_SIZE;
 #endif
     sh.map_size = pgsize + sh.arena_size + pgsize;
 
-#ifdef MAP_ANON
+#if !defined(_WIN32)
```

Comment:
There seems to be a variety of `defined(_WIN32)` and `!defined(_WIN32)`.  I don't know if making them consistent would be better or not.  Thoughts anyone?